### DISCUSSION TITLE
Welcome to rtcwPro Discussions!

### DISCUSSION THREAD

**Discussion by natelo** [rtcwmp-com/rtcwPro#295] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by v1rkes** [comment]
hello 
i have a question about r_mode -2.
when i'm in game i set /r_mode -2 or i write it in my config autoexec 
it works fine when i'm in game but when i re-open the game is not fullscreen. it's  supposed to be desktop resolution and im in 1920x1080
if you can make some test about it. or if it's my problem.
